### DISCUSSION TITLE
Extract DB to spreadsheet

### DISCUSSION THREAD

**Discussion by Fish31520** [Nachtzuster/BirdNET-Pi#224] [Q&A]
Hi there,

I'm wondering if there's a way to extract the database in order to use it in Calc or Excel (my goal would be to have birds' names in column A, dates in row 1 and to fill it with the number of detection for every day) ?

Thanks a lot !

Jean

**Comment by alexbelgium** [comment]
Hi, even easier than that : all detections are written in a BirdDB.txt csv file by default. You just need to open it and sort the columns.

**Reply by Fish31520** [reply]
Very nice, it's perfect for me, thanks a lot !
